egov-indexer/open-hearing-indexer.yml

```
@@ -53,4 +53,6 @@ ServiceMaps:
               - inJsonPath: $.hearingTypeOrder
                 outJsonPath: $.Data.hearingDetails.hearingTypeOrder
               - inJsonPath: $.hearingDurationInMillis
-                outJsonPath: $.Data.hearingDetails.hearingDurationInMillis
\ No newline at end of file
+                outJsonPath: $.Data.hearingDetails.hearingDurationInMillis
+              - inJsonPath: $.serialNumber
+                outJsonPath: $.Data.hearingDetails.serialNumber
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)